test/scheduler.js

```diff
@@ -1,4 +1,7 @@
+import test from 'tape'
+
 import { take, put, race } from '../src/effects'
+import { asap, flush, suspend } from '../src/internal/scheduler'
```

Comment:
previous tests in this file were not directly dependant on `scheduler`, is it neccessary to expose this implementation details here?